types/gulp-postcss/gulp-postcss-tests.ts

```diff
@@ -0,0 +1,42 @@
+import * as gulp from 'gulp';
+import * as postcss from 'gulp-postcss';
```

Comment:
Since these are callable entities, it isn't legal in ES2015 to import and call/construct them with a namespace import like `import * as foo from 'foo'`, so could you please convert them to either

```ts
import foo = require('foo')
```

or the following with `esModuleInterop` turned on

```ts
import foo from 'foo';
```